examples/11_fs-router/src/pages/_root.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactNode } from 'react';
+
+export default function Root({ children }: { children: ReactNode }) {
+  return (
+    <html>
+      <head>
+        <title>Waku</title>
```

Comment:
we don't need this.

Reply:
Do you mean the title or the entire file?

I think it's helpful to have one example using the `_root.tsx` file at least

Reply:
I meant the title tag.